modules/oadp-features.adoc

```diff
@@ -9,13 +9,26 @@
 OpenShift API for Data Protection (OADP) supports the following features:
 
 Backup::
-You can back up all resources in your cluster or you can filter the resources by type, namespace, or label.
+You can use OADP to back up all customer applications on the OpenShift Platform, or you can filter the resources by type, namespace, or label.
 +
 OADP backs up Kubernetes objects and internal images by saving them as an archive file on object storage. OADP backs up persistent volumes (PVs) by creating snapshots with the native cloud snapshot API or with the Container Storage Interface (CSI). For cloud providers that do not support snapshots, OADP backs up resources and PV data with Restic.
 
++
+[NOTE]
+====
+OADP will not successfully back up operators or etcd. There are a variety of ways to customize a backup and restore to avoid backing up inappropriate resources via namespaces or labels.
```

Comment:
```suggestion
OADP may not successfully restore operators out of the box. Operators generated resources with OwnerReferences to another operator custom resource should generally be excluded and let the operator recreate them for restore to be successful.
```

Reply:
I think you could use the same note for restore as well.. "will not restore " is a bit strong imo.